specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-10-01-preview/cancelEligibility.json

```diff
@@ -0,0 +1,180 @@
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "CancelEligibilityModel"
+    }
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/cancelEligibility": {
+      "post": {
+        "tags": [
+          "CancelEligibilityModel"
+        ],
+        "operationId": "CancelEligibilityModel_CancelEligibility",
+        "x-ms-examples": {
+          "cancelEligibilitySubscription": {
+            "$ref": "./examples/cancelEligibilitySubscription.json"
+          }
+        },
+        "description": "The operation to check whether the subscription is eligibile to cancel.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/CancelEligibilityResponse"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CancelEligibilityResponse": {
+      "type": "object",
+      "properties": {
+        "isEligibleToCancel": {
+          "type": "boolean",
+          "description": "Is eligible to cancel"
+        }
+      },
+      "description": "Cancel Eligibility Response",
+      "required": [
+        "isEligibleToCancel"
+      ]
+    },
+    "ErrorResponseBody": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Reasons"
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "Reason": {
+      "type": "object",
+      "properties": {
+        "SeverityLevel": {
+          "type": "string"
+        },
+        "ErrorCode": {
+          "type": "string"
+        },
+        "ErrorMessage": {
+          "type": "string"
+        },
+        "ResourceLockOwners": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-typespec-name": "string[]"
+        },
+        "Resources": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-typespec-name": "string[]"
+        }
+      },
+      "required": [
+        "SeverityLevel",
+        "ErrorCode",
+        "ErrorMessage"
+      ]
+    },
```

Comment:
Do you need to define this yourself, or can you use (or inherit from) the standard ErrorResponse definition in common-types?

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json)